### ISSUE TITLE
The conversion of Tensorflow model to FP16 model was problematic in GPU environment testing

### ISSUE THREAD

**Issue by fanxiaochen456** [openvinotoolkit/openvino#12025]
Hello, I use OpenVINO's model quantization script: Mo_tf.py converts the. Pb file with the accuracy of FP32 into the model of FP16, which is based on tensorflow1.14 environment. After the model is successfully converted and the inference process is successfully completed in the Inter CPU environment, the inference result is consistent with the result before quantization. However, WHEN I changed the Device of reasoning code to GPU, and tested the FP16 model in the environment of Inter GPU, his reasoning result was quite different from that of CPU. This problem still occurred when I changed the new version of OpenVINO for model quantization, may I ask why?

**Comment by fanxiaochen456** [comment]
![image](https://user-images.githubusercontent.com/101849755/176810967-566569a4-a360-41ea-b814-656076c5f1b1.png)
![image](https://user-images.githubusercontent.com/101849755/176811033-8ae752ce-901b-4571-a24f-8e2b17216e25.png)
These are my two results.

**Comment by brmarkus** [comment]
Can you provide more details about your environment, please, which SW, which HW do you use, in which versions, please?

Have you just converted your TF model into IR with FP32 and FP16 precision, or have you also used quantization?
How have you called the model-optimizer, can you share the command line?

Would it be possible to provide more details about the used model? What architecture/topology does it have? What is it doing, what input and output parameter(s)?

How does the code look like?
Is swapping/reversing the color channels (like RGB to BGR) needed - done via model-optimizer and/or application?

For using the GPU for inferencing OpenVINO uses OpenCL to compile OpenCL kernels; have you installed the NEO-OpenCL-driver via the OpenVINO installation script `sudo -E ./install_NEO_OCL_driver.sh`, or have you installed a different version manually?
Do you use an OpenCL-kernel-cache, and might need to clean it? (via ENV variable `cl_cache_dir` pointing to a writable folder)?
Which SOC, which CPU and GPU do you use? Do you use a discrete GPU?

**Comment by fanxiaochen456** [comment]
@brmarkus Hello, first of all, the code I use is AdvancedEAST. For the text detection algorithm of slanted text, its algorithm environment is tensorflow1.14.0+Keras2.2.5, and the model saved by Keras is adopted: .h5, then I use a script to solidify the.h5 model into a.pb model, and then I use the command: Python mo_tf.py --input_model my_model.pb --input_shape [1,736,736,3] --output east_detect/concat --data_type The IR model of FP16 is converted by FP16, and the model is not quantized before being converted to IR model. Which OpenCL do you mean? May I ask whether it needs to be installed on Windows platform? The configuration of Windows platform is Inter CPU630, which supports CPU and Inter GPU. The reverse channel has not been used for the time being. Um, do the installation scripts you mentioned need to be configured for Windows platform? I haven't found those scripts. The GPU used is Inter's GPU

**Comment by fanxiaochen456** [comment]
@brmarkus Hello, may I ask what needs to be installed if you want to use Inter GPU test environment on Windows platform?

**Comment by brmarkus** [comment]
Starting-point is this - assuming you installed OpenVINO via an offline/online-installer:
https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_windows_header.html
(watch-out for the version in the upper-right corner; you might not use "latest", but an older version)

Then go here:
https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_windows.html#optional-steps

and select GPU, to reach here:
https://docs.openvino.ai/latest/openvino_docs_install_guides_configurations_for_intel_gpu.html#gpu-guide-windows`

**Comment by fanxiaochen456** [comment]
@brmarkus The install_dependencies you mentioned were not found in the Windows download package. I installed the inference package only.

**Comment by brmarkus** [comment]
You have not mentioned MS-Windows before, not in your initial post... ;-)
For MS-Windows, please check
https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_windows_header.html
(watch-out for the version in the upper-right corner; you might not use "latest", but an older version)

Then go here:
https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_windows.html#optional-steps

and select GPU, to reach here:
[https://docs.openvino.ai/latest/openvino_docs_install_guides_configurations_for_intel_gpu.html#gpu-guide-windows`](https://docs.openvino.ai/latest/openvino_docs_install_guides_configurations_for_intel_gpu.html#gpu-guide-windows%60)

**Comment by jgespino** [comment]
Closing, please re-open if additional assistance is needed.
